README.md

```diff
@@ -108,26 +110,81 @@ In Device Simulator Express, you can use keyboard to interact with the device:
   - [MicroPython documentation](https://microbit-micropython.readthedocs.io/en/latest/)
   - [BBC micro:bit examples on the official micro:bit website](https://microbit.org/projects/make-it-code-it/?filters=python)
 
-### Keybindings
-- Push Button `A for A, B for B, C for A & B`
-- Refresh the simulator: `SHIFT + R`
-- Run the simulator: `SHIFT + F`
+### Keyboard Shortcuts
+- Push Button: <kbd>A</kbd> for Button A, <kbd>B</kbd> for Button B, <kbd>C</kbd> for Buttons A & B
+- Refresh the simulator: <kbd>Shift</kbd> + <kbd>R</kbd>
+- Run the simulator: <kbd>Shift</kbd> + <kbd>F</kbd>
+
+## Adafruit CLUE Simulator
+NOTE: This simulator is hidden under the feature flag. See below on how to enable the feature flag.

```

Comment:
Should we call it "preview mode flag" because that is what the setting is called?
```suggestion
NOTE: This simulator is hidden under the preview mode flag. See below on how to enable the preview mode flag.

```